### PR TITLE
Update on the footer

### PR DESCRIPTION
**Pull request by Iseost**
Design on the footer is updated.
Also added links to the other pages.

Needs to be add to rest of the pages.

### CODE REVIEW

**Comment by annechh** [comment]
Could we speed up this PR for merge please?

**Comment by EliNygard** [comment]
You have not requested a review from QA. We haven't gotten a notification about this and that's why our SM has not assigned it yet.

**Comment by annechh** [comment]
Oh right. Understandable if you have not get pinged :D Are you on it now then Eli ?

**Comment by EliNygard** [comment]
Great! Thank you! I only looked at this PR because I was curious since it was about the footer, and then I saw your comment :D I'm on it now!

File: index.html
Comment (EliNygard) [inline index.html:732]: Is it correct that Section 7 goes out? I see it still is in the deign file.

**Review by EliNygard — CHANGES_REQUESTED** [review]
One question before ready to merge :)

**Pull request merged**
